### DISCUSSION TITLE
Copy Artifact to new location, but creating a new 'folder' in that new location.

### DISCUSSION THREAD

**Discussion by spisone** [anancarv/python-artifactory#181] [General]
I am using the Copy artifact to new location.  Works great.  But what I am looking to do now is to copy the artifact to a "folder" inside that new location.  The new location was already created as a local repo in Artifactory, as test_repo.  When I copy the artifact, I want to create a "Ver_1.0" folder inside that new location.
For example, I want to copy test.bin to a test_repo/Ver_1.0/ location.
I tried using Update a Repository, but I get errors.
The test_repo will stay the same, but every time I copy to this, I want to copy the file to a Ver_1.0, then Ver_2.0, etc.  Any help is appreciative.  Thanks

**Comment by anancarv** [comment]
Hey @spisone,
This behavior is not supported by Artifactory (See [API](https://jfrog.com/help/r/jfrog-rest-apis/copy-item)). I suggest you to manually add the suffix` Ver_X` when copying one artifact multiple times into the same folder.
